vars/createChangeSet.groovy

```diff
@@ -291,20 +302,22 @@ def collectNestedStacks(changes) {
 }
 
 @NonCPS
-def printChanges(changeList,stackName) {
+def printChanges(changeList,stackName, expandedDetails) {
   def border = null
   def title = null
   def changeString = ""
   String[] headers = ['Operation', 'LogicalResourceId', 'ResourceType', 'Replace', 'Details']
   ArrayList data = []
 
   changeList.each { changes ->
-    border = "\n+" + "-".multiply(7) + "+" + "-".multiply(changes.getAt(0).stack.length() + 2) + "+"
-    changeString += "${border}\n| Stack | ${changes.getAt(0).stack} |${border}" + "\n"
-    data = []
     if (changes) {
+      border = "\n+" + "-".multiply(7) + "+" + "-".multiply(changes.getAt(0).stack.length() + 2) + "+"
```

Comment:
If I didn't move these lines inside the 'if', I get a null pointer exception for nested stacks without changes.